src/test/java/org/broadinstitute/hellbender/tools/CountVariantsIntegrationTest.java

```diff
@@ -16,20 +18,55 @@ public String getTestedClassName() {
     }
 
     @Test(dataProvider = "filenames")
-    public void testCountBases(String fileIn) throws Exception {
+    public void testCountVariants(String fileIn, String moreArgs, long expectedCount) throws Exception {
         final File ORIG_FILE = new File(getTestDataDir(), fileIn);
-        final String[] args = new String[]{
-            "--variant" ,  ORIG_FILE.getAbsolutePath(),
-
-        };
-        final Object res = this.runCommandLine(args);
-        Assert.assertEquals(res, 26l);
+        ArgumentsBuilder ab = new ArgumentsBuilder();
+        ab.add("--variant " + ORIG_FILE.getAbsolutePath());
+        ab.add(moreArgs);
+        final Object res = this.runCommandLine(ab.getArgsArray());
+        Assert.assertEquals(res, expectedCount);
     }
 
     @DataProvider(name="filenames")
     public Object[][] filenames() {
```

Comment:
Use better names for your `DataProviders` -- eg., here use something like `countVariantsVCFInputs`